core/model/application.go

```diff
@@ -380,6 +380,7 @@ type IdentityProviderSetting struct {
 	EmailField      string `bson:"email_field"`
 	RolesField      string `bson:"roles_field"`
 	GroupsField     string `bson:"groups_field"`
+	Farpa           string `bson:"farpa"`
```

Comment:
Please follow the same approach as other fields with Field sufix. Also it is not Farpa but Ferpa.